packages/logrotate/build

```diff
@@ -21,9 +21,9 @@ nomail
 
 /var/lib/dcos/mesos/log/* {
     olddir /var/lib/dcos/mesos/log/archive
-    maxsize 2000k
-    daily
-    rotate 7
+    maxsize 256M
```

Comment:
Mesos, Marathon, Zookeeper among other things all share that `/var/lib. The max size can do for this is probably 1-2GB (2GB being about the max). Note 2GB of logs is a lot of lines of text compressed.